pyproject.toml

```diff
@@ -52,6 +52,7 @@ sqlalchemy = "==1.4.40"
 werkzeug = "^3.0.2"
 faker = "^24.4.0"
 setuptools = "^69.2.0"
+sqlalchemy-utils = "^0.41.2"
```

Comment:
Since this is only used for our tests, it might be better to place this in the `dev` dependencies sections below, but we'd need to verify it still loads properly locally and in CI/CD.

Reply:
100% agree.

Reply:
Moved it to the dev [tool.poetry.group.dev.dependencies] dependencies.

Reply:
Awesome, thanks!